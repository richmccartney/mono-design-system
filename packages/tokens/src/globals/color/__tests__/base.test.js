const baseColorTokens = require('../base.json');

describe('Base color tokens', () => {
  test(': matches snapshot', () => {
    expect(baseColorTokens).toMatchSnapshot();
  });
});
