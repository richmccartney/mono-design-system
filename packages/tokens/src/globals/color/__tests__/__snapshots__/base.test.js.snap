// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base color tokens : matches snapshot 1`] = `
Object {
  "color": Object {
    "action": Object {
      "primary": Object {
        "value": "{color.primary.value}",
      },
      "secondary": Object {
        "value": "{color.secondary.value}",
      },
      "tertiary": Object {
        "value": "{color.base.gray.value}",
      },
    },
    "base": Object {
      "black": Object {
        "value": "#000000",
      },
      "gray": Object {
        "value": "#EFF1EE",
      },
      "green": Object {
        "value": "#00FF00",
      },
      "red": Object {
        "value": "#FF0000",
      },
      "white": Object {
        "value": "#FFFFFF",
      },
    },
    "primary": Object {
      "value": "{color.brand.primary.value}",
    },
    "secondary": Object {
      "value": "{color.brand.secondary.value}",
    },
  },
}
`;
